zulipterminal/ui_tools/views.py

```diff
@@ -315,12 +315,15 @@ def __init__(self, topics_btn_list: List[Any], view: Any,
         self.stream_button = stream_button
         self.list_box = urwid.ListBox(self.log)
         self.topic_search_box = PanelSearchBox(self, 'SEARCH_TOPICS')
+        self.header_list = urwid.Pile([self.stream_button,
+                                       urwid.Divider('-'),
```

Comment:
I'm not sure if we need these dashed lines here? Solid lines might be cleaner?

Reply:
Sure. Solid lines look better also.